server/modules/service-discovery/consul/consulCatalog.js

```diff
@@ -39,7 +39,17 @@ function getAllNodes(environment) {
 
 function getNode(environment, nodeName) {
   assert(nodeName, 'nodeName is required');
-  return executeConsul(environment, clientInstance => clientInstance.catalog.node.services(nodeName));
+  return executeConsul(environment, clientInstance => clientInstance.catalog.node.services(nodeName))
+    .then((nodes) => {
+      if (!nodes || !nodes.Services) {
```

Comment:
In the case where `nodes.Services` is falsey, should we return `null` or `nodes` with `nodes.Services = []`?